lib/Router.js

```diff
@@ -234,7 +235,10 @@ Router.prototype.handle = function(request, connection) {
 
         if (props.length) {
             route.handlers[method].filter(function(handler) {
-                return _.intersection(request.getProps(), handler.props).length;
+                var topLevelProps = props.map(function(prop) {
+                    return prop.split('.')[0];
+                });
+                return _.intersection(topLevelProps, handler.props).length;
```

Comment:
So this makes sure that at least the top-level props never throw a "missing" error, right? But the lower level ones will from what I can see. Do we want to allow this inconsistency?